pennylane/operation.py

```diff
@@ -302,20 +302,30 @@ class ParameterFrequenciesUndefinedError(OperatorPropertyUndefined):
 class WiresEnum(IntEnum):
     """Integer enumeration class
     to represent the number of wires
-    an operation acts on"""
+    an operation acts on.
+
+    .. warning:: This class is
+    """
```

Comment:
Incomplete warning.

Reply:
```suggestion
    """
```
Will add in second PR.